host/cluster-down.yaml

```diff
@@ -5,6 +5,13 @@ metadata:
   name: cluster-down
 spec:
   uri: https://raw.githubusercontent.com/replicatedhq/troubleshoot-specs/main/host/cluster-down.yaml
+  collectors:
```

Comment:
The `cluster-down.yaml` spec is usually used when the cluster is down. Given that the `copyFromHost` collector runs inside a pod within the cluster, this will most likely always fail.

Once https://github.com/replicatedhq/troubleshoot/issues/881 is implemented, we can add a collector here.

Reply:
This spec is meant to be used when the cluster is down and we suggest that to users when we point them to [this community post](https://community.replicated.com/t/kurl-how-can-i-generate-a-support-bundle-when-my-kubernetes-cluster-is-down/911).

In cases where we have a cluster running, and we want to troubleshoot host related problems as well, we can combine two specs like so

```sh
./support-bundle cluster-down.yaml default-kurl.yaml
```